DQM/L1TMonitor/src/L1TStage2RegionalMuonCandComp.cc

```diff
@@ -100,10 +122,10 @@ void L1TStage2RegionalMuonCandComp::bookHistograms(DQMStore::IBooker& ibooker, c
   // This needs to come after the calls to setBinLabel.
   errorSummaryNum->getTH1F()->GetXaxis()->SetCanExtend(false);
 
-  errorSummaryDen = ibooker.book1D("errorSummaryDen", "denominators", 14, 1, 15); // range to match bin numbering
+  errorSummaryDen = ibooker.book1D("errorSummaryDen", "denominators", nbins-3, 1, nbins-2); // range to match bin numbering
```

Comment:
This should have the same binning as errorSummaryNum. Use the same variables to set the binning.